client/src/js/references/components/Detail/Targets/Targets.js

```diff
@@ -88,7 +94,7 @@ export class Targets extends React.Component {
                     <p>Manage the allowable sequence targets for this barcode reference.</p>
                 </TargetsHeader>
 
-                <div>{targetComponents}</div>
+                <div>{targetComponents.length ? targetComponents : none}</div>
```

Comment:
This can be changed to the below block when the previous comment is fixed:
```js
{targetComponents}
{noneFound}
```
There is no need for an extra wrapping `div` here.